lib/setopt.c

```diff
@@ -1252,6 +1252,14 @@ static CURLcode setopt(struct Curl_easy *data, CURLoption option,
                                &data->set.str[STRING_PASSWORD]);
     break;
 
+  case CURLOPT_SSL_SESSION_FILE:
+    /*
+     * user:ssl-session-file to use in the operation
+     */
+    result = Curl_setstropt(&data->set.str[STRING_SSL_SESSION_FILE],
+                            va_arg(param, char *));
```

Comment:
I think it should return an error here if a non-implemented TLS backend is used.

Reply:
fixed.